src/scripts/getmeza.sh

```diff
@@ -166,11 +166,24 @@ case ${distro} in
 		;;
 esac
 
-
-# if /opt/meza doesn't exist, clone into and use master branch (which is the
-# default, but should we make this configurable?)
+# Meza repository URL and branch can be set in your system shell as environment variables so as to work locally with 
+# your preferred setup and/or test changes to this file in development without needing to alter this file explicitly. 
+#
+# Use 
+# export MEZA_REPOSITORY_URL='https://github.com/freephile/meza.git'
+# in your shell to override the default repo URL set here
+MEZA_REPOSITORY_URL="${MEZA_REPOSITORY_URL:-https://github.com/nasa/meza.git}"
```

Comment:
You can call getmeza.sh using environment variables to override the default values like this

```bash
MEZA_REPOSITORY_URL=https://github.com/freephile/meza.git \
MEZA_BRANCH_NAME=REL1_39  \
/opt/meza/src/scripts/getmeza.sh
```